website/docs/reference/resource-configs/schema.md

```diff
@@ -44,6 +44,36 @@ This would result in the generated relation being located in the `mappings` sche
 </File>
 </TabItem>
 
+<TabItem value="snapshots" label="Snapshots">
+
+<VersionBlock lastVersion="1.8">
+
```

Comment:
Should we just not include snapshots in the table for versions before 1.9?

Reply:
Unfortunately, the version blocks don't work for an entire tab, just the content within it

Reply:
bummer!